src/components/Board/Board.js

```diff
@@ -56,16 +93,36 @@ const Board = () => {
         />
         <h1>Bill Splitter</h1>
       </header>
+     
       <NewItem onAddItems={addItemHandler} />
-      <Items datas={items} remove={removeItem} />
+      
+      {/* Check if items are available before rendering Items component */}
+      <div className="items">
```

Comment:
I don't think this is necessary. It's already handled on the Items component. so you just need to call `<Items datas={items} remove={removeItem} />`.

Reply:
Got it